packages/saber/example/pages/go_bath.md

```diff
@@ -0,0 +1,15 @@
+---
+title: go_bath
+layout: post
+---
+
+She went to bathroom.
+
+```javascript {source:"../assets/go_bath.js"}
+```
+
+```go {source:"../assets/go_bath.go"}
+```
+
+```css {source:"../assets/go_bath.css"}
+```
```

Comment:
that's actually a weird syntax, how about inventing a new one using Rust macro style:

```rust
import_snippet!("../sample.js")
```

And create a markdown-it plugin to process tokens like `macro_name!($expr)`

Reply:
I find a better syntax:

```
<import-snippet path="../sample.js" />
```

Reply:
agree, it's a nice idea to express reference to code with one built-in tag/element. We can get better semantic attribute, as it could used in **ALL** transformer. Implementation now can only support default markdown transformer of saber

Reply:
This looks better:

````markdown
```{external:true}
../sample.js
```
````